app/services/cross_strategy_coordinator.py

```
@@ -0,0 +1,597 @@
+            # High severity: Skip or significantly modify
+            conflict = high_severity_conflicts[0]
+            
+            if conflict["type"] == PositionConflict.OPPOSING_DIRECTIONS:
+                # Choose higher confidence signal, skip lower confidence
+                conflicting_strategy = conflict.get("conflicting_strategy", "")
+                
+                coordination.coordination_action = "skip"
+                coordination.conflict_resolution = f"Skipped due to opposing position from {conflicting_strategy}"
+                
+                self.logger.warning(
+                    f"🚫 Signal skipped due to opposing direction conflict",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)